app/Resources/views/inside-elife-article.html.twig

```diff
@@ -2,7 +2,7 @@
 
 {% set hypothesis = true %}
 {% set hypothesis = true %}
```

Comment:
We may as well adjust this at the same time. No need to set the same variable twice.

Reply:
The duplicated variable has been removed.